jms/org.wso2.carbon.transport.jms/src/main/java/org/wso2/carbon/transport/jms/utils/JMSUtils.java

```diff
@@ -58,35 +55,7 @@ public static Destination lookupDestination(Context context, String destinationN
         if (null == destinationName) {
             return null;
         }
-        try {
             return JMSUtils.lookup(context, Destination.class, destinationName);
-        } catch (NameNotFoundException e) {
```

Comment:
Currently, the NameNotFoundException is handled from here and there was an implementation to handle it from the caller of this method(JMSConnectionFactory -> createDestination). But the since this implementation was used, the other catch block was never invoked. Therefore after a discussion with @shafreenAnfar we decided to remove this catch block and let the catch block in the caller of this method handle the exception. The way the caller of this method handles the exception solves the issue #2044 .